### PR TITLE
feat: add "new" command and connect with Studio

### PR DESCRIPTION
**Pull request by fmvilas**
**Description**
It adds the `asyncapi new` command and optionally connects it with a local instance of Studio. See #78 for more info.

**Related issue(s)**
#78 

- [x] Implement `asyncapi start studio` command.
- [x] https://github.com/asyncapi/studio/pull/141

### CODE REVIEW

File: src/commands/new.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    } else {
+      fileName = DEFAULT_ASYNCAPI_FILE_NAME;
+    }
+
+    await this.createAsyncapiFile(fileName);
+    if (openStudio) {startStudio(fileName);}
+  }
+
+  async createAsyncapiFile(fileName:string) {
+    const defaultAsyncapiFile = await readFile(resolve(__dirname, '../../assets/asyncapi.yaml'), { encoding: 'utf8' });
```
[the remark this answers is not in the document]
Comment (boyney123) [reply src/commands/new.ts:82]: Will raise a new card for this idea, maybe allow people to select protocol etc

[the remark this answers is not in the document]
Comment (fmvilas) [reply src/commands/new.ts:82]: Yeah, ideally there should be something like an onboarding questionnaire. And/or delegate this to Studio UI.

File: asyncapi.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+asyncapi: '2.2.0'
```
[the remark this answers is not in the document]
Comment (fmvilas) [reply asyncapi.yaml]: I don't know why you're seeing this file. It's not there anymore 🤔

**Comment by boyney123** [comment]
Just been through it, mostly LGTM @fmvilas 

A couple of thoughts:

- Maybe you should update the README with the new command here as part of this PR? so we document the feature and how to use it?

- I guess when you opt into the studio in the CLI the websocket stuff will also happen behind the scenes. Wonder if this should be optional or not... the socket/live changes. Let's leave it for now maybe... but wonder if people would find value not having it synced.... dunno.

**Comment by fmvilas** [comment]
> Maybe you should update the README with the new command here as part of this PR? so we document the feature and how to use it?

Oh yeah, definitely. Going to update it. Thanks.

> I guess when you opt into the studio in the CLI the websocket stuff will also happen behind the scenes. Wonder if this should be optional or not... the socket/live changes. Let's leave it for now maybe... but wonder if people would find value not having it synced.... dunno.

Let's find out. If people ask for this feature (or lack) they can add it later.

File: src/models/Studio.ts [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        type: 'file:loaded',
+        code,
+      }));
+      sendQueuedMessages();
+    });
+
+    socket.on('message', (event: string) => {
+      try {
+        const json:any = JSON.parse(event);
+        if (json.type === 'file:update') {
```
[the remark this answers is not in the document]
Comment (boyney123) [reply src/models/Studio.ts:70]: Cool, I think this is done 👍

**Review by boyney123 — APPROVED** [review]

**Review by boyney123 — APPROVED** [review]

**Pull request merged**